DetectorDescription/Core/interface/DDDivision.h

```diff
@@ -91,45 +92,35 @@ class DDDivision : public DDBase<DDName, DDI::Division*>
    */
   DDDivision(const DDName & name,
              const DDLogicalPart & parent,
-	     const DDAxes axis,
-	     const int nReplicas,
-	     const double width,
-	     const double offset );
-
+	     DDAxes axis,
```

Comment:
@davidlange6 - these are modifications from clan tidy. I would assume that a next PR with performance-unnecessary-value-param fixes it. However, it does not for some reason.